### ISSUE TITLE
[iPhone] Reflected wave animation at wrong angle

### ISSUE THREAD

**Issue by samreid** [phetsims/bending-light#368]
![image](https://cloud.githubusercontent.com/assets/679486/22562913/befc0410-e93b-11e6-830c-b59d2e12be03.png)

**Comment by samreid** [comment]
This looks like it was also buggy on 1.1.1 http://phet.colorado.edu/sims/html/bending-light/1.1.1/bending-light_en.html but the wave was too gray to notice it.

Flagging for high priority since this is very buggy on our main site now.  Should we revert until this is fixed?  Try to quickly fix the animation angle?  @ariel-phet what do you recommend?

**Comment by mattpen** [comment]
@samreid - I can recreate this bug on iPad 2 Mini, iOS 10.2.1 on both versions 1.1.2 and 1.1.1

(Apologies for the poor screenshot-fu)

![win_20170202_11_38_18_pro](https://cloud.githubusercontent.com/assets/11747676/22563203/dd046820-e93c-11e6-8b68-be762c2c7e11.jpg)

**Comment by ariel-phet** [comment]
@samreid revert

**Comment by phet-steele** [comment]
@samreid talking about #359?

**Comment by samreid** [comment]
Yes, @phet-steele that seems correct.

@jonathanolson can you please publish a maintenance revert?

**Comment by phet-steele** [comment]
@samreid @ariel-phet why would a revert be necessary here? It is still a problem in 1.1.1, 1.1.2 didn't introduce a new bug, and clearly from the screenshots in #359, the wave was NOT always too gray to notice.

**Comment by jonathanolson** [comment]
Revert complete, bumped to 1.1.3 (same behavior as 1.1.1).

**Comment by ariel-phet** [comment]
@phet-steele 1.1.2 made the bug more obvious

**Comment by jonathanolson** [comment]
Unassigned, please reassign if you want me to fix or assist with a fix for the underlying problem.

**Comment by kathy-phet** [comment]
Note, this bug does not appear in webgl=false version, only webgl version.

**Comment by kathy-phet** [comment]
@jonathanolson - When there is a lull due to testing or other things you are waiting on, can you take a look at this error since it is only in WebGL code?

**Comment by kathy-phet** [comment]
It actually seems like it might be something general in WebGL node that should be corrected, and just manifests itself here for some reason.

**Comment by ariel-phet** [comment]
Marking at low priority since this is a "as time allows" bug and has not been reported to phethelp

**Comment by phet-steele** [comment]
Closing as a duplicate of #359, relevant info has been moved.
